cargo-shuttle/src/init.rs

```diff
@@ -669,7 +670,12 @@ pub fn cargo_shuttle_init(path: PathBuf, framework: Framework) -> Result<()> {
     let mut dependencies = Table::new();
 
     // Set "shuttle-service" version to `[dependencies]` table
-    let manifest_path = find(Some(path.as_path())).unwrap();
+    let config = cargo::util::config::Config::default().unwrap();
```

Comment:
Nice!

I'm just not sure if `cargo_shuttle_init` is the correct function to update though. I think this function needs to be updated or removed (which is better)

https://github.com/shuttle-hq/shuttle/blob/79ff57e1a2e712748803d20e7aa51b1f9dbf72af/cargo-shuttle/src/lib.rs#L210-L214

Reply:
Have you run it already or shall I do it??

Reply:
https://docs.rs/project-root/latest/src/project_root/lib.rs.html#19-34

I think replacing existing function with this one will do.

Reply:
I'll leave the testing to you :smile: